src/Tribe/Editor/Objects/Editor_Object_Interface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * The API provided by an editor object.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Editory\Objects
+ */
+
+namespace Tribe\Events\Editor\Objects;
+
+/**
+ * Class Editor_Object_Interface
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Editory\Objects
+ */
+interface Editor_Object_Interface {
+
+	/**
+	 * Returns the editor object data in the format required by the block editor.
```

Comment:
* :no_entry_sign: Doc comment for parameter "$key" missing
* :no_entry_sign: Doc comment for parameter "$default" missing

<sub>hash: d78f262eda5a06f55ffdfdb652bbf6cf</sub>